Formula/unxip.rb

```diff
@@ -0,0 +1,24 @@
+class Unxip < Formula
+  desc "Fast Xcode unarchiver"
+  homepage "https://github.com/saagarjha/unxip"
+  url "https://github.com/saagarjha/unxip.git", tag: "v1.1.3", revision: "b3ce1ab4728d4173390c97eddd28e821ad9e2974"
+  license "LGPL-3.0-only"
+  head "https://github.com/saagarjha/unxip.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on macos: :monterey
```

Comment:
Since `unxip` uses APIs that aren't available on Linux, it won't make sense for it to be installable on Linux. Can I just add a separate line around here that says:

```
  depends_on :macos
```
? Or is there a preferred way to do this?

Reply:
yeah, separate line `depends_on :macos`

Reply:
I think that when I added this line, CI created a [tests job for Big Sur](https://github.com/Homebrew/homebrew-core/actions/runs/3415668383/jobs/5685109396) when it hadn't before when I had only `depends_on macos: :monterey`. Is that possible?